pkg/sidecar/sidecar_test.go

```diff
@@ -0,0 +1,93 @@
+package sidecar
+
+import (
+	"context"
+	"math/rand"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/testground/sdk-go/network"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+// Configures the default network
+func TestNetworkInitialize(t *testing.T) {
+	reactor, err := NewMockReactor()
+	if err != nil {
+		t.Fatal(err)
+	}
+	ctx := context.Background()
+	r := reactor.(*MockReactor)
+
+	go func() {
+		if err := r.Handle(ctx, handler); err != nil {
+			t.Error(err)
+		}
+	}()
```

Comment:
This is indeed a bit verbose. We could at least extract into a function, as it is used in all test method, but could be done later as well.